docs/BUILD.rst

```diff
@@ -0,0 +1,86 @@
+##########
+Lago build
+##########
+
+Lago allows to build / configure VM disks during init stage.
+In the init file, the key ``build`` should be added to each disk that needs to be configured.
```

Comment:
I would add here a small note/warning that this happens behind-the-back of the OS and  before the VM starts, thus should be used with care.